tests/ui/unused-crate-deps/test-use-not-ok.rs

```diff
@@ -0,0 +1,11 @@
+//@ edition:2018
+//@ check-pass
+//@ aux-crate:bar=bar.rs
+//@ compile-flags:--test
+
+fn main() {}
+
+#[test]
+fn test_bar() {
+    assert_eq!(bar::BAR, "bar");
```

Comment:
This test seems unrelated to namespaced crates, right?

Reply:
Yeah, no idea what this test is, seems to be incomplete.